pkg/sort/sort.go

```diff
@@ -291,6 +291,17 @@ func Sort(desc, nullsLast, hasNull bool, os []int64, vec *vector.Vector) {
 		} else {
 			genericSort(col, os, blockidGreater)
 		}
+	case types.T_json:
+		data, area := vector.MustVarlenaRawData(vec)
+		col := struct {
+			data []types.Varlena
+			area []byte
+		}{data: data, area: area}
+		if !desc {
+			genericSort(col, os, varlenaLess)
```

Comment:
I am not convinced varlenaLess is the right method for sorting json.     Sorting on json object is probably not well defined so we don't care.   We want at least sort string and number correctly.    Esp, do we know we sort number correctly?   Very large number, negative, float/int, etc,

Reply:
yes, I have done some test, which showed that the solution is wrong, I will find another solution, thanks!